fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1049,6 +1068,15 @@ void MultiVersionApi::setNetworkOption(FDBNetworkOptions::Option option, Optiona
 }
 
 void MultiVersionApi::setNetworkOptionInternal(FDBNetworkOptions::Option option, Optional<StringRef> value) {
+	auto itr = FDBNetworkOptions::optionInfo.find(option);
+	if(itr != FDBNetworkOptions::optionInfo.end()) {
+		TraceEvent("SetNetworkOption").detail("Option", itr->second.name).detail("Value", printable(value));
```

Comment:
Hm, a few of our network options, including, say, setting the TLS password, could conceivably contain sensitive information. Do we want to log the value in that case?

Reply:
Probably not

Reply:
We could potentially mark options as being sensitive in fdb.options and then not log the values for sensitive options.

Reply:
That sounds reasonable enough for me. I guess there's one caveat, which is that if adding that option is optional, then some people might forget if the value is sensitive (and "false" is the default) or not sensitive (and "true" is the default), but also having it on all of them sounds annoying. I'm not sure what the best solution is.

That's assuming that just logging the key isn't good enough, which it might not be for debugging things like a misconfigured multi-version client directory, which would seem to be something we would want to get out of this.

Reply:
Yeah, I agree that defaulting to false would be a risk for exposing sensitive information. 

It's possible that a lot of the benefit comes from just knowing which options are being set, so omitting the value may be ok. I'll take a look over the options and think about it.

Reply:
I opted to remove the logging of values and add specific logging where it seemed useful. I did add logging of knob values under the assumption that we aren't keeping sensitive data there.